OpenRA.Mods.Common/Widgets/ViewportControllerWidget.cs

```diff
@@ -195,6 +195,9 @@ public override void Draw()
 				if (Game.Settings.Game.ViewportEdgeScroll && Game.Renderer.WindowHasInputFocus)
 					edgeDirections = CheckForDirections();
 
+				if (Ui.KeyboardFocusWidget != null || world.Paused)
+					keyboardDirections = ScrollDirection.None;
```

Comment:
The player should still be able to scroll when the game is paused. That's a separate issue #18464

Reply:
Sorry, fixed